utils/avro-components-utils/src/main/scala/pl/touk/nussknacker/engine/avro/encode/JsonSchemaOutputValidator.scala

```diff
@@ -0,0 +1,103 @@
+package pl.touk.nussknacker.engine.avro.encode
+import cats.data.Validated.condNel
+import cats.data.{NonEmptyList, Validated, ValidatedNel}
+import cats.implicits._
+import com.typesafe.scalalogging.LazyLogging
+import org.everit.json.schema.{ObjectSchema, Schema}
+import pl.touk.nussknacker.engine.api.NodeId
+import pl.touk.nussknacker.engine.api.typed.typing._
+import pl.touk.nussknacker.engine.json.JsonSchemaTypeDefinitionExtractor
+import pl.touk.nussknacker.engine.util.output._
+
+import scala.language.implicitConversions
+
+private[encode] case class JsonSchemaExpected(schema: Schema) extends OutputValidatorExpected {
+  override def expected: String = JsonSchemaOutputValidatorPrinter.print(schema)
+}
+
+object JsonSchemaOutputValidator {
+  private[encode] val SimplePath = "Data" //todo: check why is it needed, and if we can get rid of it.
```

Comment:
The question is: can here be just string / int instead of object `{}`?  "Data" is used where we can't build path to field, because we got here simple primitive value.

Reply:
Ok, I see. I will change it to "Value"

Reply:
You can change it in avro as well :) Value seems to be better name.